app/assets/javascripts/discourse/app/lib/to-markdown.js

```diff
@@ -379,6 +379,10 @@ export class Tag {
         }
 
         if (cssClass?.includes("emoji")) {
+          if (cssClass?.includes("user-status")) {
```

Comment:
I believe `?` is not necessary.

Reply:
better yet, this function should not use `class` but `classList` instead - so it doesn't accidentally partial-match `emoji` and `user-status` substrings 😃

Reply:
I agree that `classList` would be better here, but it's not available, because `this.element` here is our own Element class:

https://github.com/discourse/discourse/blob/8f9933d1dd6597687c6d7b6ec95290e75d3cfed5/app/assets/javascripts/discourse/app/lib/to-markdown.js#L698

We can pass `classList` to this class for sure, so it will be available, but let's keep this fix short and simple. I've dropped the second `?` for now.